assets/pages/contributions-landing/contributionsLanding.scss

```diff
@@ -1,507 +1,508 @@
-		height: auto;
-		width: auto;
-		background-color: gu-colour(error);
-		padding-top: $gu-v-spacing * 2.5;
-		padding-bottom: $gu-v-spacing * 2.5;
-
-		span {
-			margin-right: $gu-h-spacing;
-		}
-	}
-
-	// ----- Legal Copy ----- //
-
-	.contributions-legal {
-		background-color: darken(#fff, 5%);
-	}
-
-	.contributions-legal__content {
-		background-color: #fff;
-		padding-top: $gu-v-spacing * 3;
-		padding-bottom: $gu-v-spacing;
-
-		@include mq($from: desktop) {
-			padding-top: $gu-v-spacing * 2;
-		}
-
-		@include mq($from: desktop, $until: leftCol) {
-			padding-bottom: $gu-v-spacing * 9;
-		}
-
-		@include mq($from: leftCol) {
-			padding-bottom: $gu-v-spacing * 12;
-		}
-
-	}
-
-	.component-contrib-legal {
-		@include mq($from: tablet) {
-			width: 620px;
-		}
-
-		@include mq($from: desktop) {
-			width: 540px;
-		}
-
-		@include mq($from: leftCol) {
-			width: 670px;
-		}
-
-		@include mq($from: wide) {
-			width: 700px;
-		}
-	}
+    }
+  }
 
-}
+  .component-cta-link {
+    padding-top: $gu-v-spacing;
+    padding-bottom: $gu-v-spacing;
+    margin-top: $gu-v-spacing * 4;
+
+    @include mq($from: desktop) {
+      // Fix for IE.
+      svg {
+        left: 245px;
+      }
+    }
+
+    @include mq($from: wide) {
+      // Fix for IE.
+      svg {
+        left: 325px;
+      }
+    }
+
+    svg {
+      height: 35%;
+      top: 15px;
+      right: 12px;
+    }
+  }
+
+  // ----- PayPal button & error message ----- //
+
+  .component-paypal-contribution-button {
+    padding-top: 17px;
+    padding-bottom: 17px;
+    cursor: pointer;
+    display: block;
+    font-family: $gu-text-sans-web;
+    font-size: 14px;
+    font-weight: bold;
+    height: auto;
+    width: 100%;
+    line-height: 180%;
+    margin-top: $gu-v-spacing;
+    position: relative;
+    text-align: left;
+    text-decoration: none;
+    vertical-align: bottom;
+    margin-bottom: $gu-v-spacing;
+  }
+
+  .component-error-message {
+    height: auto;
+    width: auto;
+    background-color: gu-colour(error);
+    padding-top: $gu-v-spacing * 2.5;
+    padding-bottom: $gu-v-spacing * 2.5;
+
+    span {
+      margin-right: $gu-h-spacing;
+    }
+  }
+
+  // ----- Legal Copy ----- //
+
+  .contributions-legal {
+    background-color: darken(#fff, 5%);
+  }
+
+  .contributions-legal__content {
+    background-color: #fff;
+    padding-top: $gu-v-spacing * 3;
+    padding-bottom: $gu-v-spacing;
+
+    @include mq($from: desktop) {
+      padding-top: $gu-v-spacing * 2;
+    }
+
+    @include mq($from: desktop, $until: leftCol) {
+      padding-bottom: $gu-v-spacing * 9;
+    }
+
+    @include mq($from: leftCol) {
+      padding-bottom: $gu-v-spacing * 12;
+    }
+
+  }
+
+  .component-contrib-legal {
+    @include mq($from: tablet) {
+      width: 620px;
+    }
+
+    @include mq($from: desktop) {
+      width: 540px;
+    }
+
+    @include mq($from: leftCol) {
+      width: 670px;
+    }
+
+    @include mq($from: wide) {
+      width: 700px;
+    }
+  }
+
+}
```

Comment:
Has anything changed in this file aside from adding `#contributions-landing-page-au` and the indentation?

Reply:
No, it is just my opinionated IntelliJ Idea